.gitignore

```diff
@@ -19,3 +19,4 @@ examples/**/obj/
 public.crt
 go_build_operator_
 operator.iml
+.run/
```

Comment:
Not sure why this is needed...

Reply:
because in that place I save my IntelliJ configuration but I don't think this is used by all. So maybe better to keep it out.